File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/library/repository/permissions/storage/read/request/IStorageReadPermissionsRequestNotificationBuilder.java
package com.lasthopesoftware.bluewater.client.library.repository.permissions.storage.read.request;

import android.app.Notification;

/**
 * Created by david on 7/3/16.
 */
public interface IStorageReadPermissionsRequestNotificationBuilder {
	Notification buildReadPermissionsRequestNotification(int libraryId);
}

File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/stored/library/permissions/read/IStorageReadPermissionsRequestNotificationBuilder.kt
package com.lasthopesoftware.bluewater.client.stored.library.permissions.read

import android.app.Notification
import com.lasthopesoftware.bluewater.client.browsing.library.repository.LibraryId

/**
 * Created by david on 7/3/16.
 */
interface IStorageReadPermissionsRequestNotificationBuilder {
    fun buildReadPermissionsRequestNotification(libraryId: LibraryId): Notification?
}
